### ISSUE TITLE
jme3test.asset.TestManyLocators crashes due to missing assets

### ISSUE THREAD

**Issue by stephengold** [jMonkeyEngine/jmonkeyengine#982]
Some files required to run `TestManyLocators` have moved. Paths need to be updated.

Currently (in `master` branch at 3.3-6680) it crashes with the following stack trace:
```
Exception in thread "main" com.jme3.asset.AssetLoadException: Failed to open zip file: town.zip
	at com.jme3.asset.plugins.ZipLocator.setRootPath(ZipLocator.java:79)
	at com.jme3.asset.ImplHandler$ImplThreadLocal.initialValue(ImplHandler.java:120)
	at java.lang.ThreadLocal.setInitialValue(ThreadLocal.java:180)
	at java.lang.ThreadLocal.get(ThreadLocal.java:170)
	at com.jme3.asset.ImplHandler.tryLocate(ImplHandler.java:178)
	at com.jme3.asset.DesktopAssetManager.locateAsset(DesktopAssetManager.java:191)
	at jme3test.asset.TestManyLocators.main(TestManyLocators.java:59)
Caused by: java.io.FileNotFoundException: town.zip (The system cannot find the file specified)
	at java.util.zip.ZipFile.open(Native Method)
	at java.util.zip.ZipFile.<init>(ZipFile.java:225)
	at java.util.zip.ZipFile.<init>(ZipFile.java:155)
	at com.jme3.asset.plugins.ZipLocator.setRootPath(ZipLocator.java:77)
	... 6 more
```

**Comment by Ali-RS** [comment]
Once I created a pr #565 to add town.zip to jme testdata. You can read the reason for the objection there.

**Comment by Ali-RS** [comment]
Note, town.zip along with some other assets are is also available in wiki
https://github.com/jMonkeyEngine/wiki/tree/master/src/docs/resources/Scenes

**Comment by stephengold** [comment]
That's fine for other tests. However, accessing `town.zip` via HTTP would defeat the purpose behind including it in `TestManyLocators`, which is to test/demonstrate a `ZipLocator`.

**Comment by stephengold** [comment]
Fixed in `master` branch at hash 000ac95 .

I copied the `town.zip` file from the Google Code archives to the `jme3-examples` sub-project. (No effect on `jme3-testdata`.)

**Comment by Ali-RS** [comment]
Hmm, still have the issue.


```
Exception in thread "main" com.jme3.asset.AssetLoadException: Failed to open zip file: town.zip
	at com.jme3.asset.plugins.ZipLocator.setRootPath(ZipLocator.java:81)
	at com.jme3.asset.ImplHandler$ImplThreadLocal.initialValue(ImplHandler.java:120)
	at java.lang.ThreadLocal.setInitialValue(ThreadLocal.java:180)
	at java.lang.ThreadLocal.get(ThreadLocal.java:170)
	at com.jme3.asset.ImplHandler.tryLocate(ImplHandler.java:173)
	at com.jme3.asset.DesktopAssetManager.locateAsset(DesktopAssetManager.java:203)
	at jme3test.asset.TestManyLocators.main(TestManyLocators.java:59)
Caused by: java.io.FileNotFoundException: town.zip (No such file or directory)
	at java.util.zip.ZipFile.open(Native Method)
	at java.util.zip.ZipFile.<init>(ZipFile.java:225)
	at java.util.zip.ZipFile.<init>(ZipFile.java:155)
	at com.jme3.asset.plugins.ZipLocator.setRootPath(ZipLocator.java:79)
	... 6 more
```

I am on Linux
